pkgs/top-level/all-packages.nix

```
@@ -8130,6 +8130,8 @@ with pkgs;
 
   gitlab-workhorse = callPackage ../applications/version-management/gitlab/gitlab-workhorse { };
 
+  gitlab-elasticsearch-worker = callPackage ../applications/version-management/gitlab/gitlab-elasticsearch-worker { };
```

The upstream repository is called `gitlab-elasticsearch-indexer`

```suggestion
  gitlab-elasticsearch-indexer = callPackage ../applications/version-management/gitlab/gitlab-elasticsearch-indexer { };
```